discovery-provider/plugins/pedalboard/apps/relay/src/routes/relay.ts

```diff
@@ -0,0 +1,23 @@
+import App from "basekit/src/app";
+import { RelayRequestType, RelayResponseType } from "../types/relay";
+import { SharedData } from "..";
+import { relayTransaction } from "../txRelay";
+
+export const relayHandler = async (
+  app: App<SharedData>,
+  req: RelayRequestType
+): Promise<RelayResponseType> => {
+  try {
+    const { receipt } = await relayTransaction(app, req);
+    return {
+      receipt: {
+        blockHash: receipt.blockHash,
+        blockNumber: receipt.blockNumber,
+      },
+    };
+  } catch (e) {
+    // return useful error back to caller from here
+    console.error("relay error = ", e);
+    throw e;
```

Comment:
do we want to throw here? if so are we catching anywhere? may be best to just console.error the issue and not disrupt the flow of things if something goes wrong

Reply:
Yes, this will be caught by fastify and coerced into the right error it thinks. There's a future ticket to improve this and make the errors more readable.